packages/forma-36-codemod/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@contentful/f36-codemod",
   "description": "Forma 36 Codemod",
-  "version": "4.5.0",
+  "version": "5.0.0-alpha.0",
```

Comment:
We need to update the deps as well here, we're still pointing to icons v4.

Reply:
good point, actually I think we don't need the icons dependency here 🤔